cmake/Install.cmake

```diff
@@ -152,3 +152,20 @@ function(ly_install_run_code CODE)
     )
 
 endfunction()
+
+#! ly_install_run_script: specifies path to script to be added to the install process (will run at install time)
+#
+# \notes: 
+#  - refer to cmake's install(SCRIPT documentation for more information
+#
+function(ly_install_run_script SCRIPT)
```

Comment:
I think it is awkward to name the input argument SCRIPT after the name of the `keyword` script.

Visually wise I think it should at least be lowercase `script` or some other name.

Reply:
I used the same pattern ly_install_run_code() is using above. I can change both.

Reply:
You are right `ly_install_run_code` does have a paramteer with the name of `CODE`.
It is not a big deal if you leave it as is.